setup.py

```diff
@@ -70,6 +84,22 @@ def build_extensions(self):
     if not os.path.exists(join("pyopenjtalk", "openjtalk" + ext)):
         raise RuntimeError("Cython is required to generate C++ code")
 
+# make openmp available
+system = platform.system()
+if system == "Windows":
+    extra_compile_args = []
+    extra_link_args = ["/openmp"]
+elif system == "Linux":
+    extra_compile_args = ["-fopenmp"]
+    extra_link_args = ["-fopenmp"]
+elif system == "Darwin":
+    os.system("brew install libomp")
+    extra_compile_args = ["-Xpreprocessor", "-fopenmp"]
+    extra_link_args = ["-L/usr/local/lib", "-lomp"]
```

Comment:
Better to avoid hard-code

Reply:
Maybe we can add an option for it and let user decide where to link against?

Reply:
Actually the linker will search all path for this lib, so we can just remove "-L/usr/local/lib"

Reply:
An environmental variable (with default) works okay for example. https://github.com/r9y9/pyopenjtalk/blob/d4c59cdf6644626e4e659462fab91858cec1fd9c/setup.py#L221